app/views/katello/api/v2/subscriptions/base.json.rabl

```diff
@@ -0,0 +1,19 @@
+object @resource
+
+attributes :cp_id => :id
```

Comment:
Perhaps we should move this to database ID and also include the cp_id since we are using `with_identifier` and supporting both, I believe we did this with the content types as well.